.github/workflows/docker-publish-njsPC-linux.yml

```diff
@@ -11,71 +11,22 @@ jobs:
   build-and-push:
     runs-on: ubuntu-latest
```

Comment:
Is this still valid with the 3 linux builds?

Reply:
Yes, this is just the platform that the runner uses, not the resulting images.